File: src/main/java/alfheim/common/item/lens/LensMessanger.java
package alfheim.common.item.lens;

import net.minecraft.item.ItemStack;
import vazkii.botania.api.mana.BurstProperties;
import vazkii.botania.common.item.lens.Lens;

public class LensMessanger extends Lens {
	
	@Override
	public void apply(ItemStack stack, BurstProperties props) {
		props.maxMana /= 5;
		props.ticksBeforeManaLoss *= 3;
		props.motionModifier *= 3F;
	}
}

File: src/main/java/alfheim/common/item/lens/LensMessanger.kt
package alfheim.common.item.lens

import net.minecraft.item.ItemStack
import vazkii.botania.api.mana.BurstProperties
import vazkii.botania.common.item.lens.Lens

class LensMessanger: Lens() {
	
	override fun apply(stack: ItemStack?, props: BurstProperties) {
		props.maxMana /= 5
		props.ticksBeforeManaLoss *= 3
		props.motionModifier *= 3f
	}
}